gui/src/components/NestedRoutes.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { ReactNode } from "react";
+import { Router, useLocation, useRouter } from "wouter";
```

Comment:
weren't you dropping wouter in favor of something else?

Reply:
Yeah, I decided to keep it. React Router keeps breaking its API, so we might as well stick with this.